contracts/bepro/Network_v2.sol

```diff
@@ -437,32 +391,26 @@ contract Network_v2 is Governed, ReentrancyGuard {
 
     /// @dev enable users to fund a bounty
     function fundBounty(uint256 id, uint256 fundingAmount) external payable {
-        // _bountyExists(id);
+
         _isFundingRequest(id, true);
         _isInDraft(id, true);
         _isNotCanceled(id);
 
         Bounty storage bounty = bounties[id];
         require(bounty.funded == false, "F1");
         require(bounty.tokenAmount < bounty.fundingAmount, "F2");
-        require(bounty.tokenAmount < bounty.tokenAmount.add(fundingAmount), "F3");
 
         bounty.funding.push(Benefactor(msg.sender, fundingAmount, block.timestamp));
 
         bounty.tokenAmount = bounty.tokenAmount.add(fundingAmount);
         bounty.funded = bounty.fundingAmount == bounty.tokenAmount;
 
-        //ERC20 erc20 = ERC20(bounty.transactional);
-        // uint256 settlerAmount = fundingAmount * calculatePercentPerTenK(bounty.settlerTokenRatio);
         require(ERC20(bounty.transactional).transferFrom(msg.sender, address(this), fundingAmount), "F3");
-        // require(settlerToken.transferFrom(msg.sender, address(this), settlerAmount), "F4");
-
-        // totalSettlerLocked = totalSettlerLocked.add(settlerAmount);
+        emit BountyFunded(id, bounty.funded);
```

Comment:
why not sending the amount and the address of the sender ?

Reply:
I didn't see a need, since we should be updating / needing the whole bounty value when we receive this event we should make a getBounty instead

Reply:
In that case the event consumer will have to read the bounty from the blockchain and know how much the bounty has in funds. i think it could be handy to have it